WORKSPACE

```diff
@@ -1,12 +1,20 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 rules_scala_version = "b537bddc58a77318b34165812a0311ef52806318"
 
-http_archive(
+
+git_repository(
     name = "io_bazel_rules_scala",
-    strip_prefix = "rules_scala-%s" % rules_scala_version,
-    type = "zip",
-    url = "https://github.com/bazelbuild/rules_scala/archive/%s.zip" % rules_scala_version,
+    remote = "https://github.com/bazelbuild/rules_scala",
+    commit = "ca5a7acff4ff630f68f58b8e01e8c25dbf908fb7" # HEAD as of 2019-05-13, update this as needed
+)
+
+http_archive(
```

Comment:
Protobuf here too.

Reply:
Same as decco and we need to do these changes on multicrypto as well